vmdb/app/models/miq_provision_microsoft/cloning.rb

```diff
@@ -88,6 +88,15 @@ def memory_ps_script
     end
   end
 
+  def cpu_ps_script
+    cpu_script = ""
+    cpu_script << "-CPUCount #{cpu_count} "
```

Comment:
@gmcculloug - personally I think it is harder to read than what is already there plus the small change suggested by @blomquisg like this:

def cpu_ps_script
    cpu_script = "-CPUCount #{cpu_count} "
    cpu_script << "-CPUReserve #{cpu_reserve} " unless cpu_reserve.nil?
    cpu_script << "-CPUMaximumPercent #{cpu_max} " unless cpu_max.nil?
    cpu_script
  end

@blomquisg - what do you think?